package.json

```diff
@@ -23,13 +23,13 @@
   },
   "preferGlobal": true,
   "scripts": {
-    "test": "node npm/test.js",
+    "test": "npm run test-lint && npm run test-system && npm run test-unit && npm run test-integration && npm run test-cli && npm run test-library",
     "test-system": "node npm/test-system.js",
     "test-lint": "node npm/test-lint.js",
     "test-unit": "node npm/test-unit.js",
-    "test-integration": "node npm/test-integration.js",
-    "test-cli": "node npm/test-cli.js",
-    "test-library": "node npm/test-library.js",
+    "test-integration": "nyc --temp-dir .coverage --silent node npm/test-integration.js",
```

Comment:
`nyc` reports coverage as "unknown" unless the whole file is executed with nyc as done here. @codenirvana